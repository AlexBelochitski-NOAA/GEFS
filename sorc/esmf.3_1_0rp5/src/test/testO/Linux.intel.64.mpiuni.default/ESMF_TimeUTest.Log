20190610 112232.853537 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112232.853780 INFO     PET0  Beginning Test, file ESMF_TimeUTest.F90, line          83
20190610 112232.853798 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112232.853860 INFO     PET0  PASS  Set Time Initialization Test, ESMF_TimeUTest.F90, line         111
20190610 112232.853911 INFO     PET0  PASS  Get Time Test 1, ESMF_TimeUTest.F90, line         123
20190610 112232.856387 INFO     PET0  Number of failed tests:           0
20190610 112232.856398 INFO     PET0  Ending Test, file ESMF_TimeUTest.F90, line         901
