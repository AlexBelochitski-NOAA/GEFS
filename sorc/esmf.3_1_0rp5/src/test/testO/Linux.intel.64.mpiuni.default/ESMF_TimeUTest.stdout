  Beginning Test, file ESMF_TimeUTest.F90, line          83
  NUMBER_OF_PROCESSORS           1
  PASS  Set Time Initialization Test, ESMF_TimeUTest.F90, line         111
  PASS  Get Time Test 1, ESMF_TimeUTest.F90, line         123
 startTime = 
 2004-01-29T12:17:58                                                            
                                                  
  Number of failed tests:           0
  Ending Test, file ESMF_TimeUTest.F90, line         901
